app/process/roads_bikelanes/roads_bikelanes.lua

```diff
@@ -125,7 +125,9 @@ function osm2pgsql.process_way(object)
 
   if not PathClasses[tags.highway] then
     MergeTable(results, Maxspeed(object))
-    MergeTable(results, BikelanesPresence(object, cycleways))
+    if not HighwayClasses[tags.highway] then
+      MergeTable(results, BikelanesPresence(object, cycleways))
+    end
```

Comment:
Die Logik hätte ich lieber in der BikelanesPresence.lua
- bspw. übe reinen guard recht weit oben "if autobahn then return empty presence"
- aber besser wäre noch, wie würden auch auf diesen highway classes erstmal die prozessierung laufen lassen und nur am ende dann den overwrite machen "if autobahn and nil, then "not expected" (ähnlich dem `replace all nil values with 'missing'`)